test/NaCl.Core.SimdTests/ChaCha20ScalarTests.cs

```diff
@@ -0,0 +1,347 @@
+﻿namespace NaCl.Core.SimdTests
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+
+    using FluentAssertions;
+    using Xunit;
+
+    using Base;
+    using Internal;
+    using Vectors;
+    using NaCl.Core.Base.ChaChaCore;
+    using NaCl.Core.SimdTests.Vectors;
```

Comment:
I don't think this using is needed since above it you are already specifying `using Vectors;`